core/src/test/scala/dagr/core/cmdline/DagrCoreMainTest.scala

```diff
@@ -102,6 +102,16 @@ class DagrCoreMainTest extends UnitSpec with CaptureSystemStreams with BeforeAnd
     }
   }
 
+  it should "should print the pipeline arguments if printPipelineArgs is true" in {
+    Seq(true, false).foreach { withPipelineArgsPrinter =>
+      val pipelineName = if (withPipelineArgsPrinter) nameOf(classOf[PipelineFive]) else nameOf(classOf[PipelineFour])
+      val args = Array[String](pipelineName, "--argument", "argument")
+      val (_, _, stderr, _) = testParse(args)
+      if (withPipelineArgsPrinter) stderr should include (s"$pipelineName --argument argument")
```

Comment:
Yeah, see I think we also want `--flag false` here too, so that it's recorded in case the default value changes over time.